gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java

```diff
@@ -32,6 +34,8 @@
   private static final Set<String> SCOPES = ImmutableSet.of(PUBSUB_SCOPE);
   private static final String DEFAULT_HOST = "https://pubsub.googleapis.com";
 
+  private AckDeadlineRenewParams ackDeadlineRenewParams;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'ackDeadlineRenewParams' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2634248459/issues/source?bid=3123109&fileBranchId=3319389#l37)